docs/rules/trials.md

```diff
@@ -0,0 +1,80 @@
+!!! note
+	This guide is simply a recommendation, you are not require to follow it when conducting trials
```

Comment:
Remove tab, use spaces. (Or switch to an editor that doesn't make this mistake; VS Code should help you out.)